google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProviderTest.java

```diff
@@ -782,7 +782,7 @@ public void testFromSpace() throws Exception {
 
     // getPath does not interpret the string at all.
     Path path4 = provider.getPath("gs://bucket/with/a%20percent");
-    assertThat(path4.toString()).isEqualTo("/with/a%20percent");
+    assertThat(path4.toString()).isEqualTo("gs://bucket/with/a%2520percent");
```

Comment:
original string includes: `a%20percent`
new string includes: `a%2520percent`
just wondering: where did `25` come from?

Reply:
this thing is happen because of `PercentEscaper` but now these will return `a%20percent` instead of `a%2520percent`